### ISSUE TITLE
Can not reproduce the results

### ISSUE THREAD

**Issue by lxtGH** [facebookresearch/Motionformer#4]
I run the motionformer for three different setting on Kinetics datasets. 

I run motionformer_224_16x4.yaml with batchsize 8 using 8GPU. Finally, I got 71.48 top-1 on val dataset.

I run configs/K400/joint_224_16x4.yaml. I got 76.63 on val dataset. 

I run configs/K400/divided_224_16x4.yaml. I got 76.27 on val dataset.

**Comment by JaminFong** [comment]
Me neither. I got a 77.0 top-1 accuracy on the val dataset running `K400/motionformer_224_16x4.yaml` with a total batch size of 32 on 8 GPUs.

**Comment by lxtGH** [comment]
@JaminFong Hi! What is lr schedule strategy? When i use bs 32, the test error is huge after 2 epoch training.

**Comment by JaminFong** [comment]
@lxtGH Hi, I follow the provided config exactly.

**Comment by mandelapatrick** [comment]
Hey, @lxtGH and @JaminFong, thanks for your interest in our work. Have you been able to reproduce reported results using the pre-trained checkpoints available on the repo? That should hopefully verify that your data pipeline is working. In terms of training the models, the config assumes that you are using 8 nodes (8 GPUs per node), for an effective batch size (EBS) of 32/node * 8 nodes = 256. I think with a much smaller batch-size (32 for @JaminFong or 8 for @lxtGH), I do expect some regression in performance as transformers can be sensitive to batch-size, but to get as close to a performance as mines with 8 nodes, you would have to appropriately scale the learning rate. SOLVER.BASE_LR is 1e-4 for EBS=256. @JaminFong you should try re-running with SOLVER.BASE_LR == 32/256*1e-4, and @lxtGH you should try, 8/256*1e-4 as LR. Hope this helps.

**Comment by JaminFong** [comment]
Thanks for your reply. The learning rate setting may matter.

**Comment by PeiqinZhuang** [comment]
> Thanks for your reply. The learning rate setting may matter.

Hi, have you tried to rescale your learning rate according to your batch size. Actually, the learning rate has an impact with the final result. The top1 accuracies are 77.0% and 78.3% respectively, with learning rate of 1e-4 and 32 / 256 * 1e-4 , which uses a small batch size of 32 with 8 GPUs. However, it seems there still remains a performance gap with the figure reported in that paper. By the way, I find that the evaluation process is relatively slow. Have you found this problem?

**Comment by PeiqinZhuang** [comment]
> Hey, @lxtGH and @JaminFong, thanks for your interest in our work. Have you been able to reproduce reported results using the pre-trained checkpoints available on the repo? That should hopefully verify that your data pipeline is working. In terms of training the models, the config assumes that you are using 8 nodes (8 GPUs per node), for an effective batch size (EBS) of 32/node * 8 nodes = 256. I think with a much smaller batch-size (32 for @JaminFong or 8 for @lxtGH), I do expect some regression in performance as transformers can be sensitive to batch-size, but to get as close to a performance as mines with 8 nodes, you would have to appropriately scale the learning rate. SOLVER.BASE_LR is 1e-4 for EBS=256. @JaminFong you should try re-running with SOLVER.BASE_LR == 32/256_1e-4, and @lxtGH you should try, 8/256_1e-4 as LR. Hope this helps.

Hi, I have tried to rescale the learning rate according to the batch size.  Actually, the learning rate has an impact with the final result. The top1 accuracies are 77.0% and 78.3% respectively, with learning rate of 1e-4 and 32 / 256 * 1e-4 , which uses a small batch size of 32 with 8 GPUs. However, it seems there still remains a performance gap with the figure reported in that paper. By the way, I find that the evaluation process is relatively slow. Have you found this problem?

**Comment by lxtGH** [comment]
No I abandon.
